### ISSUE TITLE
Feature Request: "Custom Command" for all selected dupes

### ISSUE THREAD

**Issue by eproskurin** [hsoft/dupeguru#562]
It would be great, if the "Custom Command" can process not only currently selected dupe, but all selected (or checked) dupes one by one.

Many thanks!

**Comment by Juraj2** [comment]
I vote for the feature as well. 
It would be very useful to let to run the custom command automatically for every selected file and not have to run it manually for every file. Custom command can be for example to custom rename the duplicate and move it to a specific location.

**Comment by glubsy** [comment]
This should not be too difficult to implement. Simply add a new placeholder variable which would hold every selected item.

**Comment by Weafyr** [comment]
Any chances for this feature being ever implemented? - Its not even user friendly, I was confused why custom command action is applied only on file that has been clicked on and not to all files as the other action does.

**Comment by glubsy** [comment]
Yes this feature could be implemented. I might end up doing it in the near future, I just have to find some free time to work on it. It's on my TODO list at least.

**Comment by arsenetar** [comment]
It is already possible to select multiple duplicates with shift/ctrl and then apply the custom command to the whole batch.  Selection != Marked (marked is when you have checked the box next to a file).

**Comment by FredWahl** [comment]
Very good idea! I haven’t used but I assumed that it worked like you suggest.
